pom.xml

```diff
@@ -21,7 +21,7 @@
 
   <groupId>io.cdap.plugins</groupId>
   <artifactId>elasticsearch-plugins</artifactId>
-  <version>1.9.0</version>
+  <version>1.9.1</version>
```

Comment:
Since this is a change of Java version being support, I think it's better to merge against develop branch rather than changing a release branch.

Reply:
This is done.